vowpalwabbit/parse_regressor.cc

```diff
@@ -18,6 +18,8 @@ using namespace std;
 #include <math.h>
 #include <algorithm>
 #include <stdarg.h>
+#include <numeric>
+#include <random>
```

Comment:
Instead of using <random> (which is not cross-platform), can you use the builtin pseudorandom numbers in rand48.h?  That's our knock-off which is cross platform.